api/controllers/game/get-list.js

```diff
@@ -4,11 +4,15 @@ module.exports = async function (req, res) {
   // HANDLE REQ.SESSION.GAME = GAME ID CASE
   sails.sockets.join(req, 'GameList');
   try {
-    const games = await gameAPI.findOpenGames();
+    const [ openGames, spectatableGames ] = await Promise.all([
```

Comment:
```suggestion
    const [ openGames, spectatableGames ] = await Promise.all([
```
I really don't like `spectatableGames` as a name, but I don't have a better alternative.

Reply:
Yeah I also don't love it and also  haven't come up with a better alternative. Maybe spectateGames?

Also is this actually a suggestion? I can't actually see the difference 😅